packages/components/src/components/textarea/textarea.stories.ts

```diff
@@ -30,400 +32,247 @@ export default {
  * This shows sd-textarea in its default state.
  */
 export const Default = {
-  args: overrideArgs([
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
+  name: 'Default',
+  args: overrideArgs([{ type: 'attribute', name: 'label', value: 'Label' }]),
   render: (args: any) => {
-    return html`<div class="w-[250px]">${generateTemplate({ args })}</div> `;
+    return html`<div class="w-[500px]">${generateTemplate({ args })}</div> `;
   }
 };
 
 /**
- * Use the `placeholder` attribute to add a placeholder.
+ * Use the `size` attribute to change the size.
  */
 
-export const Placeholders = {
-  args: overrideArgs([
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'placeholder', value: 'Placeholder' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Size = {
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-textarea value="Large" size="lg" label="Label" rows="4" spellcheck></sd-textarea>
+
+      <sd-textarea value="Medium" size="md" label="Label" rows="4" spellcheck></sd-textarea>
+
+      <sd-textarea value="Small" size="sm" label="Label" rows="4" spellcheck></sd-textarea>
+    </div>
+  `
 };
 
 /**
- * Use the disabled attribute to disable a textarea. All interaction is disabled and no events will be fired.
+ * Use the `rows` attribute to set the number of rows.
  */
-export const Disabled = {
-  parameters: {
-    controls: {
-      exclude: ['disabled']
-    }
-  },
-  args: overrideArgs([
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          constants: [{ type: 'attribute', name: 'disabled', value: true }],
-          args
-        })}
-      </div>
-    `;
-  }
+
+export const Rows = {
+  name: 'Rows',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-textarea size="lg" label="Label" rows="4" spellcheck></sd-textarea>
+
+      <sd-textarea size="lg" label="Label" rows="8" spellcheck></sd-textarea>
+
+      <sd-textarea size="lg" label="Label" rows="12" spellcheck></sd-textarea>
+    </div>
+  `
 };
 
 /**
- * Use the readonly attribute to render a textarea as readonly.  Interaction is enabled, but the textarea cannot be edited.  Events will be fired.
+ * Use the`label`attribute to give the input an accessible label.
```

Comment:
@yoezlem @coraliefeil Just to confirm, I did not add the term "code-only" since it would not make a lot of sense in Storybook. But please let me know if I should include it.